shapeshift/web
# feat: cosmos staking validators wire up

## Description

Draft PR for wire up with real validators data regarding issue https://github.com/shapeshift/web/issues/1267

## Notice




## Pull Request Type


## Issue (if applicable)

[]
(https://github.com/shapeshift/web/issues/1267)

## Risk



## Testing

- Having a wallet with real cosmos
- navigate to cosmos asset page
- verify staking values and validators are correct

## Screenshots (if applicable)

![stakingCosmos](https://user-images.githubusercontent.com/5720927/161035280-086e736b-1ba2-47f8-aa5e-f652310f1c61.png)
